Resources/BAIN/qml/BainTTestBayesianIndependentSamples.qml

```diff
@@ -47,11 +48,11 @@ Form
 			title: qsTr("Hypothesis Test")
 			name: "hypothesis"
 
-			RadioButton { text: qsTr("Equal vs. not equal")                     ; name: "groupsNotEqual" ; checked: true}
-			RadioButton { text: qsTr("Equal vs. bigger")                        ; name: "groupOneGreater" }
-			RadioButton { text: qsTr("Equal vs. smaller")                       ; name: "groupTwoGreater" }
-			RadioButton { text: qsTr("Bigger vs. smaller")                      ; name: "_4type" }
-			RadioButton { text: qsTr("Equal vs. bigger vs. smaller")            ; name: "allTypes" }
+			RadioButton { text: qsTr("Equal vs. not equal")                     ; name: "equalNotEqual" ; checked: true}
```

Comment:
That is too bad, because some bain analyses won't work anyway anymore since bain changed its model specification (anova, ancova, regression) to not include a `.`  anymore. This will break all of these .jasp file anyway...

Reply:
Well I mean you could do a simple conversion in the code between option names.
But like I said there will be a workaround available, you'll just need to make a mental note to implement that ;)

Reply:
Yes, might be nice to have some sort of template for that when the time comes